frontend/packages/dev-console/src/components/deployments/utils/deployment-utils.ts

```diff
@@ -19,7 +21,101 @@ import {
   LifecycleHookFormData,
   LifecycleHookImagestreamData,
   TriggersAndImageStreamFormData,
-} from './edit-deployment-types';
+} from './deployment-types';
+
+export const getDefaultDeploymentConfig = (namespace: string): K8sResourceKind => {
+  const defaultDeploymentConfig: K8sResourceKind = {
+    apiVersion: getAPIVersionForModel(DeploymentConfigModel),
+    kind: DeploymentConfigModel.kind,
+    metadata: {
+      namespace,
+      name: '',
+    },
+    spec: {
+      selector: {},
+      replicas: 3,
+      template: {
+        metadata: {
+          labels: {},
+        },
+        spec: {
+          containers: [
+            {
+              name: 'container',
+              image: 'image-registry.openshift-image-registry.svc:5000/openshift/httpd:latest',
+              ports: [
+                {
+                  containerPort: 8080,
+                  protocol: 'TCP',
+                },
+              ],
+            },
+          ],
+        },
+      },
+      strategy: {
+        type: DeploymentStrategyType.rollingParams,
+        rollingParams: {
+          updatePeriodSeconds: 1,
+          intervalSeconds: 1,
+          timeoutSeconds: 600,
+          maxUnavailable: '25%',
+          maxSurge: '25%',
+        },
+      },
```

Comment:
Triggers is available in D and DC, but previously they were added to DCs:

```suggestion
      },
      triggers: [
        {
          type: 'ConfigChange',
        },
      ],
```